### PR TITLE
Rename Officers to Members and more

### LINKED ISSUE #29: Create "People" tab to replace "Officers", and add content

### PR DESCRIPTION
**Pull request by mehmetoguzderin**
Closes https://github.com/sigturk/sigturk/issues/29 issue.

This PR renames the Officers tab to People, adds sections for Advisory Committee, Organization Contributors (code contributors with permission & trust to review and merge), Code Contributors (people who contribute to SIGTURK).

This PR also fixes a Makefile deficiency and creates an empty Projects tab to allow easier experimentation.

### CODE REVIEW

**Comment by mehmetoguzderin** [comment]
Preview URL: https://mehmetoguzderin.github.io/sigturk.github.io/preview/oguz-20220808-members/

**Comment by mammadhajili** [comment]
When you click people tab in your preview page, it links to

https://mehmetoguzderin.github.io/people/

and gives 404, I believe the right way to do it should be:

https://mehmetoguzderin.github.io/sigturk.github.io/preview/oguz-20220808-members/people/

**Comment by j0ma** [comment]
This sounds like a good idea to me -- calling it "People" instead of "Officers" can make us sound more approachable. Does ACL enforce any standard regarding this? 

I'm also seeing the same issue with the People tab as @mammadhajili above.

File: content/en/people/index.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+== Advisory Committee
+
+
+== Organization Contributors
+
+* Mehmet Oguz Derin link:++https://github.com/mehmetoguzderin++[(GitHub)]
+* Mammad Hajili link:++https://github.com/mammadhajili++[(GitHub)]
+* Jamshidbek Mirzakhalov link:++https://github.com/mirzakhalov++[(GitHub)]
+* Jonne Sälevä link:++https://github.com/j0ma++[(GitHub)]
```
Comment (j0ma) [inline content/en/people/index.adoc:22]: Should these have email addresses, like the president and secretary above?

File: content/en/projects/_index.adoc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Projects"
+description: "ACL Special Interest Group on Turkic Languages Projects"
+date: 2020-10-06T08:49:55+00:00
+lastmod: 2020-10-06T08:49:55+00:00
```
Comment (j0ma) [inline content/en/projects/_index.adoc]: I'm not really familiar with where this metadata is used, but should it say 2022?

File: config/_default/menus/menus.en.toml [hunk 3]
```diff
@@ -4,8 +4,8 @@
   weight = 10
 
 [[main]]
-  name = "Officers"
-  url = "/officers/"
+  name = "People"
+  url = "/people/"
```
Comment (j0ma) [inline config/_default/menus/menus.en.toml:8]: This looks like a good change assuming ACL is not imposing some standard on us where we have to call it Officers

File: content/en/people/index.adoc [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+== Code Contributors
```
Comment (j0ma) [inline content/en/people/index.adoc:24]: Is this line necessary given that it's empty? If there is content to be added, can we add it?

File: content/en/people/index.adoc [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+title: "People"
+description: "ACL Special Interest Group on Turkic Languages People"
+date: 2020-08-27T19:25:12+02:00
+lastmod: 2020-08-27T19:25:12+02:00
```
Comment (j0ma) [inline content/en/people/index.adoc]: I'm not really familiar with where this metadata is used, but should it say 2022?

File: content/en/people/index.adoc [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+draft: false
+images: []
+contributors: []
+---
+
+President:: Duygu Ataman mailto:sigturk-president@gmail.com[(email)]
+
+Secretary:: Sardana Ivanova mailto:sigturk-secretary@gmail.com[(email)]
```
Comment (j0ma) [inline content/en/people/index.adoc]: Should these have GitHub URLs like the contributors below?

File: content/en/people/index.adoc [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+
+== Advisory Committee
```
Comment (j0ma) [inline content/en/people/index.adoc:14]: Is this line necessary given that the section is (still) empty?

**Review by j0ma — COMMENTED** [review]
Great changes! My only concern is whether ACL requires that SIGs call their officers officers. Left some comments too, mostly to practice using the UI :)

File: content/en/people/index.adoc (hunk 1 above)
Comment (mehmetoguzderin) [reply content/en/people/index.adoc:22]: A good question for meeting IMO, personally leaning towards having contributors listed with their GitHub profile URL. But maybe include email for those who would like to show it?

File: content/en/projects/_index.adoc (hunk 2 above)
Comment (mehmetoguzderin) [reply content/en/projects/_index.adoc]: I will remove these, leftover

File: config/_default/menus/menus.en.toml (hunk 3 above)
Comment (mehmetoguzderin) [reply config/_default/menus/menus.en.toml:8]: @d-ataman and @mammadhajili suggestion, maybe revisit at meeting?

File: content/en/people/index.adoc (hunk 4 above)
Comment (mehmetoguzderin) [reply content/en/people/index.adoc:24]: We do not have non-org non-officer contributors yet (similar story for Advisory Committee), but request on issue was to prep these sections

File: content/en/people/index.adoc (hunk 7 above)
Comment (mehmetoguzderin) [reply content/en/people/index.adoc:14]: We do not have Advisory Committee yet (similar story for non-org non-officer contributors), but request on issue was to prep these sections

File: content/en/people/index.adoc (hunk 5 above)
Comment (mehmetoguzderin) [reply content/en/people/index.adoc]: I will remove these, leftover

File: content/en/people/index.adoc (hunk 6 above)
Comment (mehmetoguzderin) [reply content/en/people/index.adoc]: Makes sense, going to include in next commit!

**Comment by mehmetoguzderin** [comment]
@j0ma I addressed most of the comments except for the ones that would benefit from discussion at the meetings. Thanks again for your careful review!

File: content/en/people/index.adoc (hunk 1 above)
Comment (mehmetoguzderin) [reply content/en/people/index.adoc:22]: We discussed a bit at the meeting and tentatively deciding to list as little emails as possible

**Review by d-ataman — APPROVED** [review]
as a result of our last meeting we confirmed there is no standard on the officers so it is fine to approve this list of contributors.

**Pull request merged**